contentcuration/contentcuration/frontend/shared/views/details/Details.vue

```diff
@@ -231,7 +231,7 @@
             </VFlex>
             <a
               v-else
-              :href="`/channels/${channel.id}/view`"
+              :href="`/channels/${channel.id}`"
```

Comment:
Could you use `window.Urls.channel(channel.id)` instead here? That way it will stay updated if the backend URL scheme changes, or throw an error if it ceases to exist.

Reply:
This URL appears in other places too like:

https://github.com/jonboiser/content-curation/blob/2fd4cff69eb63329d3566bb7800e05ba085d22de/contentcuration/contentcuration/frontend/administration/pages/Users/UserDetails.vue#L104

Should I change that too?

Reply:
I just went ahead and changed the other `href`s

Reply:
Thanks - sorry was in meetings while you were fixing bugs!